src/js/post-911-gib-status/reducers/index.js

```diff
@@ -0,0 +1,25 @@
+import set from 'lodash/fp/set';
+
+const initialState = {
+  enrollmentData: null,
+  available: false
+};
+
+function post911GIBillStatus(state = initialState, action) {
```

Comment:
yes good point. let's go with GibStatus, to be consistent with the file names :) I'll make the change